cypress-tests/cypress/e2e/admin/pageBuilder/templates/templatePreview.cy.ts

```diff
@@ -0,0 +1,73 @@
+import { customAlphabet } from "nanoid";
+
+context("Page Builder - Templates", () => {
+    const nanoid = customAlphabet("abcdefghijklmnopqrstuvwxyz");
+    let counter = 3;
+    const pageTemplateData1 = {
+        title: "test1",
+        slug: nanoid(6),
+        description: nanoid(6),
+        tags: [],
+        layout: "static",
+        pageCategory: "static"
+    };
+    const pageTemplateData2 = {
+        title: "test2",
+        slug: nanoid(6),
+        description: nanoid(6),
+        tags: [],
+        layout: "static",
+        pageCategory: "static"
+    };
+    const pageTemplateData3 = {
+        title: "test3",
+        slug: nanoid(6),
+        description: nanoid(6),
+        tags: [],
+        layout: "static",
+        pageCategory: "static"
+    };
+
+    beforeEach(() => {
+        cy.login();
+        cy.pbDeleteAllTemplates();
+        cy.wait(200);
+        cy.createPageTemplate(pageTemplateData1);
+        cy.wait(200);
+        cy.createPageTemplate(pageTemplateData2);
+        cy.wait(200);
+        cy.createPageTemplate(pageTemplateData3);
+        cy.wait(200);
+    });
+
+    it("Should be able to create a page and view all existing templates in it", () => {
```

Comment:
This test is failing on my side.

![image](https://github.com/webiny/webiny-js/assets/5121148/1bdfad94-af20-4675-aa2f-800b23c6fdb4)